.circleci/config.yml

```diff
@@ -262,7 +268,7 @@ jobs:
       python_version: "3.8"
     steps:
       - checkout
-      - install_dependencies_dev
+      - install_dependencies_docs
```

Comment:
Did this just so that the circleci `build_docs` job would have the same deps installed by us as on RTD.

Reply:
If the goal is to have the same deps as RTD, shouldn't we install `setuptools>=45.1.0` before installing our package?

Reply:
@freddyaboulton I'd say my goal here was to have the circleci job have deps which are *closer* to whats used on the RTD docker image.

If we really wanted to go the extra mile, I think we should write a job which duplicates the RTD build in its entirety, including the docker image. We could write that automation ourselves, or we could see if there's [something in the community](https://github.com/dfm/rtds-action) which we can use.

Given that we're considering migrating to github actions early next year, I think we should punt on this question. But great thinking 😁